File: python3.9/1012.py
valores = input().split(" ")
A, B, C = [float(val) for val in valores]
TRIAN = (A * C)/2
CIRC = 3.14159 * pow(C,2)
TRAP = ((A+B)*C)/2
QUAD = B*B
RETANG = A*B
print("TRIANGULO: {:.3f}".format(TRIAN))
print("CIRCULO: {:.3f}".format(CIRC))
print("TRAPEZIO: {:.3f}".format(TRAP))
print("QUADRADO: {:.3f}".format(QUAD))
print("RETANGULO: {:.3f}".format(RETANG))

File: Codigos-C/1012.c
#include <stdio.h>
 
int main() {
 
	double A, B, C, x, y, z, p, q;
	
	scanf("%lf",&A);
	scanf("%lf",&B);
	scanf("%lf",&C);
	
	x = A*C/2;
	y = 3.14159*C*C;
	z = (A+B)*C/2;
	p = B*B;
	q = A*B;
	
	printf("TRIANGULO: %.3lf\n",x);
	printf("CIRCULO: %.3lf\n",y);
	printf("TRAPEZIO: %.3lf\n",z);
	printf("QUADRADO: %.3lf\n",p);
	printf("RETANGULO: %.3lf\n",q);
    return 0;
}
